SI/Sem.02/Tasks2/Task3.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+
+	int firstNumber, secondNumber, condition;
```

Comment:
I am not sure what is all this logic for
The actual solution is much simple
Check this solution: https://github.com/Justsvetoslavov/Introduction_to_programming/pull/22/files#diff-fb15057d198d4557d8abfe0f9118fb3ba1bde4dcdcc0b5d457cc8bfcc1ae1816